org.gramps_project.Gramps.yml

```diff
@@ -0,0 +1,87 @@
+app-id: org.gramps_project.Gramps
+# flatpak-builder will not take com.gramps-project.Gramps
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+writable-sdk: true
```

Comment:
```suggestion
```

Reply:
Without the writable Sdk, I get the following error for the GExiv2Dependency module:
```
OSError: [Errno 30] Read-only file system: '/usr/lib/python3.8/site-packages/gi/overrides/GExiv2.py'
FAILED: meson-install 
/usr/bin/meson install --no-rebuild
ninja: build stopped: subcommand failed.
Error: module gexiv2Dependency: Child process exited with code 1
```
Is there a better solution?

Reply:
By the way, I had already tried putting ensure-writable in there, but it gets the same read only file system error.
`  - name: gexiv2Dependency
    buildsystem: meson
    ensure-writable:
      - /lib/python3.8/site-packages/gi/overrides/GExiv2.py
    sources:
      - type: archive
        url:  https://download.gnome.org/sources/gexiv2/0.12/gexiv2-0.12.1.tar.xz
        sha256:  8aeafd59653ea88f6b78cb03780ee9fd61a2f993070c5f0d0976bed93ac2bd77
`

Reply:
```
python3 setup.py install --prefix=/app --root=/
```
...should make it work without `ensure-writable`.

Reply:
Thank you for responding.  This is a meson build so python won't help.  There is no setup.py.